lib/entities/release-action.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import copy from 'fast-copy'
+import { DefaultElements, ISO8601Timestamp, Link, MakeRequest } from '../common-types'
+import { wrapCollection } from '../common-utils'
+import { AsyncActionProcessingOptions, pollAsyncActionStatus } from '../methods/action'
+import enhanceWithMethods from '../enhance-with-methods'
+
+type ReleaseActionStatuses = 'created' | 'inProgress' | 'failed' | 'succeeded'
+export type ReleaseActionTypes = 'publish' | 'unpublish' | 'validate'
+
+export type ReleaseActionSysProps = {
+  id: string
+  type: 'ReleaseAction'
+  space: Link<'Space'>
+  environment: Link<'Environment'>
+  release: Link<'Release'>
+  status: ReleaseActionStatuses
+  createdBy: Link<'User'>
+  createdAt: ISO8601Timestamp
+  updatedAt: ISO8601Timestamp
+}
+
+/** The object returned by the Releases API */
+export interface ReleaseActionProps<T extends ReleaseActionTypes = any> {
+  action: T
+  sys: ReleaseActionSysProps
+}
+
+export interface ReleaseActionQueryOptions {
+  /** Find Release Actions by using a comma-separated list of Ids */
```

Comment:
Are we sure this supports a comma-separate list? Normally that would be `sys.id[in]`

Reply:
uh, yeah you are correct 🤦🏽  -- this is actually `sys.id[in]`

Reply:
:done: